### ISSUE TITLE
No keyboard input in Windows Terminal running with "runas /netonly user:<username>" flags from elevated shell

### ISSUE THREAD

**Issue by fmj9** [microsoft/terminal#9971]
### Windows Terminal version (or Windows build number)

10.0.19042.0, 1.7.1091.0

### Other Software

Guest OS Windows 10 20H2 (10.0.19042.0) inside Oracle VM VirtualBox (6.1.20 r143896). Not in domain
Host Ubuntu 20.04
Windows Terminal 1.7.1091.0

### Steps to reproduce

1. Run cmd (or Windows Terminal) as administrator
2. Run "runas /netonly /user:<username>@<domain>" wt.exe
*wt.exe linking to Windows Terminal executable
3. Windows Terminal windows opens, but keyboard input is not working

### Expected Behavior

Expecting keyboard to work in Windows Terminal's command promt.
When running not elevated cmd (or Windows Terminal) and launching Windows Terminal via "runas..." (from non elevated shell) everything works fine.

### Actual Behavior

No keyboard input in Windows Terminal launched from elevated shell via "runas...". Mouse selecting and clicking (copy/paste) works fine.

**Comment by zadjii-msft** [comment]
Huh, this is weird. This is kinda like #8045, #4448, but different in that input works fine while unelevated. @ebadger, do you have any ideas as to what might be going on here?

**Comment by ebadger** [comment]
@fmj9 Curious to know if input into notepad works if you launch it in the repo way.

for example:
Run cmd (or Windows Terminal) as administrator
Run "runas /netonly /user:%username% notepad.exe"

Does typing into notepad work?

**Comment by fmj9** [comment]
Yes, when I launch notepad via runas typing into notepad works fine.
Same when I launch cmd or powershell. The problem only occurs when I'm starting Windows Terminal this way

**Comment by mitchellrj** [comment]
I also experience this issue. I can paste by right-clicking, but no keyboard input works except for the window chrome (e.g. CTRL+SPACE). If I open the settings panel, then no keyboard input works there either.

Version: 1.13.11431.0

**Comment by ebadger** [comment]
can you check your tasklist to see if ctfmon.exe is running?

**Comment by mitchellrj** [comment]
Yes, ctfmon.exe is running.

**Comment by carlos-zamora** [comment]
@ebadger any ideas? Could this be an issue with multiple user accounts?

**Comment by Phorin** [comment]
I'm unsure if this is related, but I suspect it is...

On my work laptop image we use BeyondTrust/PowerBroker to elevate processes. When I use BeyondTrust to elevate `wt.exe`, no keyboard input is received, but only to that window. And yes, `ctfmon.exe` **is** running.

Global shortcuts like `Alt+Space` **do** work, and you can close the `Alt+Space` popup with the `ESC` key too.  (Obviously since this all runs from dwm, I'm guessing.) Absolutely nothing else works, including tab, function keys, etc. The behavior is **not** as if there's a secondary key being held down like `Alt` (e.g. pressing `Tab` does not result in an `Alt+Tab` popup).

There are no messages in the Application or System events in Event Viewer pertaining to any of this that I can find. I also was not able to quickly (in under 10 minutes) find a way to enable additional logging for Windows Terminal itself, so I cannot provide logs.

Running `cmd.exe` using BeyondTrust allows keyboard input, but then when I execute the Windows Terminal `wt.exe` executable from that elevated `cmd.exe` window, the input again does not work for the WT window. (`cmd.exe [elevated] -> wt.exe [elevated] -> no keyboard input`)

Now for the interesting part... when I run wt.exe from an ELEVATED `cmd.exe` using:

`runas /trustlevel:0x20000`

The WT input **still** does **not** work, despite the WT instance being run with non-elevated permissions (tab in WT does **not** show "Administrator: " in front of it). (`cmd.exe [elevated] -> Run as [de-escalated] wt.exe -> no keyboard input`)

It's like it's some kind of process isolation issue almost given how utterly stubborn it is being to any kind of user input in that window. We do also use the following McAfee products:

Data Exchange Layer
Drive Encryption Agent
DLP Endpoint
Application Control
Endpoint Security

As well as FireEye endpoint security.

**Comment by zadjii-msft** [comment]
MSFT:43611188 for internal tracking purposes, we'll see what the input folks have to say
